src/index.js

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable no-console */
```

Comment:
- [ ] It will be great to fix the linters errors instead of disabling some actions. The linters are there to ensure your code is clean and free of any errors.